src/scalper.ts

```diff
@@ -960,27 +964,30 @@ export class Scalper {
     const dimQty = maxBackGammaMultiple * netGamma;
     let bidDepth = 0;
     let whaleBidPrice : number;
-    for (let i = 0; i < numBids; i++){
-      bidDepth = bids.getL2(i+1)[i][1] + bidDepth;
-      if (bidDepth >= dimQty){
-        whaleBidPrice = bids.getL2(i+1)[i][0];
+    for (let i = 0; i < numBids; i++) {
+      bidDepth = bids.getL2(i + 1)[i][1] + bidDepth;
+      if (bidDepth >= dimQty) {
+        [whaleBidPrice] = bids.getL2(i + 1)[i];
```

Comment:
Were all these changes auto-generated?

Reply:
Most of them were. I did a quick pass to get about 25% of the remaining. Will do a few runs over it so that if there is an issue, easier to bisect.